rb/spec/unit/selenium/webdriver/common/interactions/pointer_actions_spec.rb

```diff
@@ -26,7 +26,7 @@ module WebDriver
       let(:builder) { W3CActionBuilder.new(bridge, mouse, keyboard) }
       let(:element) { Element.new(bridge, 'element') }
       let(:element2) { Element.new(bridge, 'element2') }
-      let(:duration) { PointerActions::DEFAULT_MOVE_DURATION }
+      let(:duration) { PointerActions::default_move_duration }
```

Comment:
since this is no longer a constant, it needs to be referenced as a method on the class that includes the `PointerActions` module. In this case, `builder`. Update this to be:
```
      let(:duration) { builder.default_move_duration }
```